wp-content/themes/citylimits/homepages/assets/less/homepage.less

```diff
@@ -421,4 +470,9 @@
       }
     } // media query for desktop
   }
+  .widget {
+    @media (max-width: 600px) {
+      margin-bottom: 20px;
```

Comment:
Referencing this comment from #112:

> on the homepage, the newsletter should only be between the top featured and "section 1". so, I don't think we need to worry about the whitespace in this case. however, we need to make sure there isn't whitespace between the special projects and footer.

Can we make this rule instead:

```less
  .widget + .widget {
    @media (max-width: 600px) {
      margin-top: 20px;
    }
  }
```

That way, it won't cause white space at the bottom of the homepage widget list.

Reply:
Updated in https://github.com/INN/umbrella-citylimits/pull/113/commits/4b459363b6c8c5bae51863da56d0c69f569ffc03